health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewChecklist.js

```
@@ -0,0 +1,316 @@
+
+const ViewChecklist = () => {
+    const { t } = useTranslation();
+    const tenantId = Digit.ULBService.getCurrentTenantId();
+    const searchParams = new URLSearchParams(location.search);
+    const campaignName = searchParams.get("campaignName");
+    const role = searchParams.get("role");
+    const rlt = searchParams.get("role");
+    const roleLocal = (!rlt.startsWith("ACCESSCONTROL_ROLES_ROLES_")) ? "ACCESSCONTROL_ROLES_ROLES_" + rlt : rlt;
+    const checklistType = searchParams.get("checklistType");
+    let clt = searchParams.get("checklistType");
+    const checklistTypeLocal = (!clt.startsWith("HCM_CHECKLIST_TYPE_")) ? "HCM_CHECKLIST_TYPE_" + clt : clt;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)